src/Styleguide/Components/__stories__/MarketInsights.story.tsx

```diff
@@ -8,7 +8,7 @@ storiesOf("Styleguide/Components", module).add("MarketInsights", () => {
   return (
     <React.Fragment>
       <Section title="Market Insights">
-        <MarketInsights artist={artistResponse} />
+        <MarketInsights artist={artistResponse as any} />
```

Comment:
I presume we're using `as any` here because we're passing a fixture?

Reply:
Yup! I added an explanation about that to the OP.